crates/swc_ecma_utils/src/function.rs

```diff
@@ -13,6 +13,14 @@ impl WrapperState {
     pub fn new(this: Option<Ident>, args: Option<Ident>) -> Self {
         Self { this, args }
     }
+    pub fn merge(&mut self, other: Self) {
+        if let Some(this) = other.this {
+            self.this = Some(this)
```

Comment:
This modifies even if `self.this` is `Some`. Is this intended?
If so, `merge` seems like a confusing name,

Reply:
Oh, this is made for `parallel`, let me remove it